demisto_sdk/commands/content_graph/interface/neo4j/queries/common.py

```diff
@@ -44,13 +44,32 @@ def node_map(properties: Dict[str, Any]) -> str:
     return f'{{{", ".join([f"{k}: {v}" for k, v in properties.items()])}}}'
 
 
-def to_neo4j_map(properties: dict) -> str:
-    properties = {
-        k: f'"{v}"' if isinstance(v, (str, Path)) else v for k, v in properties.items()
-    }
-    params_str = ", ".join(f"{k}: {v}" for k, v in properties.items())
+def to_neo4j_map(
```

Comment:
logically it's good enough (will not work for list properties like `tags` - for now, let's just mention this as a limitation in the docstring). 

I have a problem with the purpose of this method because it doesn't return just a "neo4j map" anymore.
An easy solution would be renaming it to something like `to_node_map_and_predicates()` but I'd also consider implementing something like:
```
def to_node_pattern(
    properties: dict,
    varname: str = "node",
    content_type: ContentType = ContentType.BaseContent,
) -> str:
    exact_props = {k, v for k, v in properties.items() if not isinstance(v, list)
    if multi_choice_props := {k, v for k, v in properties.items() if isinstance(v, list):
        where_clause = f" WHERE {to_predicates(varname, multi_choice_props)}"
    else:
        where_clause = ""
    return f"({varname}:{content_type}{to_neo4j_map(exact_props)}{where_clause})"
```
see https://neo4j.com/docs/cypher-manual/current/clauses/where/#node-pattern-predicates to understand the where clause part.

Another thing, it's advanced and more complex to implement, but we can improve the query performance a bit if we first check if there exists an index (see NODE_INDEX_OPTIONS in indexes.py) within the properties we sent, if so - include only them in `exact_props`, and add the rest as predicates.